src/components/Perils/PerilModal/PerilModal.tsx

```diff
@@ -254,21 +254,6 @@ export const PerilModal: React.FC<PerilModalProps & ModalProps> = (props) => {
                   </CoverageListItem>
                 ))}
               </CoverageList>
-
-              {currentPeril.exceptions.length > 0 && (
-                <CoverageList>
-                  <CoverageListTitle>
-                    {props.story?.content.peril_modal_exceptions_title ??
-                      'Exceptions'}
-                  </CoverageListTitle>
-                  {currentPeril.exceptions.map((text) => (
-                    <CoverageListItem key={text}>
-                      <Minus size="0.75rem" />
-                      {text}
-                    </CoverageListItem>
-                  ))}
-                </CoverageList>
```

Comment:
we're skipping the exceptions? Why? Can we do that (as in like - is it legal?) ?

Reply:
Carl got feedback from a few people that we shouldn't show it on the market web, since it might convert less and not other insurers do that. But in the web onboarding we should show it of course

Reply:
Alrgihty, just double checking 👌